### ISSUE TITLE
Error exporting FBX on windows

### ISSUE THREAD

**Issue by thomthom** [thomthom/cities-skylines-sketchup-tools#4]
If the user haven't used the normal FBX exporter before using the extension it fails when adjusting the settings.

```
#<Win32::Registry::Error: The system cannot find the file specified.>
C:/Program Files/SketchUp/SketchUp 2015/Tools/RubyStdLib/win32/registry.rb:385:in `open'
C:/Program Files/SketchUp/SketchUp 2015/Tools/RubyStdLib/win32/registry.rb:496:in `open'
C:/Users/tykho/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/tt_citites_skylines/exporter.rb:138:in `win32_write_settings'
C:/Users/tykho/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/tt_citites_skylines/exporter.rb:176:in `set_fbx_exporter_settings'
C:/Users/tykho/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/tt_citites_skylines/exporter.rb:34:in `export_asset'
C:/Users/tykho/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/tt_citites_skylines/core.rb:22:in `block in <module:CitiesSkylinesTools>'
SketchUp:1:in `call'
```

Need to make sure the FBX exporter key exists first.

**Comment by Mistro11** [comment]
You mean use the regular FBX within a SU session or in general? Until this gets fixed, should I do an arbitrary fbx export before using the plugin?

**Comment by thomthom** [comment]
Yes - doing a normal export from File > Export into FBX should create the appropriate registry keys which need to exist in order to modify them.

**Comment by thomthom** [comment]
Fixed in 7ee784c1f37860b9b89fc03fa2b348738264a7ab
